tests/ga/test_exthandlers.py

```diff
@@ -580,3 +581,19 @@ def capture_process_output(stdout_file, stderr_file):
             output = self.ext_handler_instance.launch_command(command)
 
         self.assertIn("[stderr]\nCannot read stdout/stderr:", output)
+
+    def test_it_should_contain_all_helper_environment_variables(self):
+
+        helper_env_vars = {ExtCommandEnvVariable.ExtensionSeqNumber: self.ext_handler_instance.get_seq_no(),
+                           ExtCommandEnvVariable.ExtensionPath: self.tmp_dir,
+                           ExtCommandEnvVariable.ExtensionVersion: self.ext_handler_instance.ext_handler.properties.version}
+
+        command = """
+            printenv | grep -E '(%s)'
+        """ % '|'.join(helper_env_vars.keys())
+
+        test_file = self.create_script('printHelperEnvironments.sh', command)
+        output = self.ext_handler_instance.launch_command(test_file)
+
+        for helper_var in helper_env_vars:
+            self.assertIn("%s=%s" % (helper_var, helper_env_vars[helper_var]), output)
```

Comment:
Thanks for adding the test. 

Just a qq: I assume this test won't fail when a new Env. variable is added to the list (I see you are only checking for the env variables mentioned in `helper_env_vars`). Is there a way to make that happen as well?